frontend/src/lib/api.ts

```diff
@@ -848,12 +848,13 @@ class ApiRequest {
     public integrationGoogleAdsConversionActions(
         id: IntegrationType['id'],
         customerId: string,
+        parentId: string,
         teamId?: TeamType['id']
     ): ApiRequest {
         return this.integrations(teamId)
             .addPathComponent(id)
             .addPathComponent('google_conversion_actions')
-            .withQueryString({ customerId })
+            .withQueryString({ customerId, parentId })
     }
```

Comment:
Nit: This could be nicer if parentId and customerId were just part of a "params" object